### DISCUSSION TITLE
One more question for "capacitor init"

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [ionic-team/capacitor#7336] [Ideas]
So today following the guide I noticed one of the steps was to run `npx cap init`

It asked 2 questions, the App Name and the Package ID

_It also asked about creating an account on Ionic (No idea how this is relevant here)_

One thing it didn't ask and it assumed was the `webDir` where it filled in by itself with `dist` 

Shouldn't this be a question during the init process? In my case my web app builds to `build` so it would be nice if this was also prompted during the init process.

**Comment by MrAmericanMike** [comment]
Adding to this as I continue reading the guide, I notice that it mentions this later

```
npx cap sync will copy your built web application, by default www, to your native project and install the native projects dependencies.

Info
You can customize what folder is copied over by modifying the webDir variable in your [Capacitor Config](https://capacitorjs.com/docs/config) file that is created during npx cap init.
```

But doesn't seam accurate, it says by default it will use `www` but the default was `dist`. Unless I'm misunderstanding something.

**Reply by jcesarmobile** [reply]
It tries to guess which folder would be the `webDir` based on the framework/build tools you use, and if it doesn't get a "result" then it uses www. So in your case I guess it detected one of the frameworks/build tools that use dist, but it might not be using dist anymore or you changed it to something else.

Which framework/build tools do you use?

**Reply by MrAmericanMike** [reply]
I see. I guess it detected Sveltekit in my case, which I believe uses `dist` by default, but my `svelte.config.js` specifies the usage of the adapter static and the `build` folder and the output.

```js
import adapter from "@sveltejs/adapter-static";
import { vitePreprocess } from "@sveltejs/vite-plugin-svelte";

/** @type {import("@sveltejs/kit").Config} */
const config = {
	preprocess: vitePreprocess(),
	kit: {
		appDir: "app",
		files: {
			appTemplate: "src/index.html",
			assets: "src/public"
		},
		adapter: adapter({
			pages: "build",
			assets: "build",
			fallback: "index.html",
			precompress: false
		})
	}
};

export default config;

```

I guess the tool making a "guess" is a good thing, but not when we may be using different values and not the defaults.

Maybe it would be a good idea to inform the user with something like:

```
It looks like you are using Svelte/Sveltekit: Is the output dir 'dist' ok, or you want to specify a different one?
```

I guess something like that could help the init process be more friendly.
(Just an idea)

Thanks for the response. I guess this could be closed if no changes are planned.

**Reply by jcesarmobile** [reply]
ah, yeah, there is an issue open for sveltekit
https://github.com/ionic-team/capacitor/issues/6150
